### DISCUSSION TITLE
Obsidian callouts styling vs. ITS callouts & admonitions styling

### DISCUSSION THREAD

**Discussion by Jnosh** [SlRvb/Obsidian--ITS-Theme#176] [Q&A] [answered]
Admonitions from the admonition plugin and ITS specific callouts (e.g. `[!recite]`) appear to use ITS specific styling but regular Obsidian callouts (e.g. `[!danger]`) seem to use mostly the default Obsidian styling.

Is this deliberate, and if so is there a way to enable the admonition styling for the Obsidian callouts as well? Or is this just a bug/missing feature? Or am I simply holding it wrong...

**Comment by SlRvb** [comment, marked as answer]
It is deliberate, but the ITS beta and v1 will have the option for applying the original styling with a toggle in the style settings.

**Reply by Jnosh** [reply]
Perfect, thank you for all the hard work on this great theme!
